/usr/local/src/conda/python-3.9.19/Objects/unicodeobject.c: 4985 - 4991
--------------------------------------------------------------------------------

